docs/Style-Guide.md

```diff
@@ -0,0 +1,173 @@
+# Fastify Style Guide
+
+
+## Welcome
+
+Welcome to *Fastify Style Guide*. This guide is here to provide you with a conventional writing style for users writing developer documentation for our Open Source framework. Each topic is precise and well explained to help you write documentation users can easily understand and implement.
+
+
+<br>
+<br>
+
+## Who is this guide for?
+
+This guide is for anyone who loves to build with Fastify or want to contribute to our documentation. You don't need to be an expert in writing technical documentation. Our guideline is here to help you.
+<br>
+
+Visit the [contribute](https://www.fastify.io/contribute) page on our website or read the [CONTRIBUTE.MD](https://github.com/fastify/fastify/blob/master/CONTRIBUTING.md) on Github to join our fellow Open Sourcers.
+
+<br>
+<br>
+
+## Before you write
+
+You need to know the following:
+
+* JavaScript
+* Node.js
+* Git
+* Github
+* Markdown
+* HTTP
+* NPM 
+
+<br>
```

Comment:
I think none of these `<br>` are needed. Why are you adding them?

Reply:
I added `<br>`  to separate each topic so that I can add more information to them later when I need to. They will be removed once I'm done. 

Would you like me to delete them from the repo?

Reply:
It's fine to keep them as placeholders, but definitely remove them before merging this pr. Thanks!